examples/hotrod/services/customer/database.go

```diff
@@ -77,7 +76,6 @@ func (d *database) Get(ctx context.Context, customerID int) (*Customer, error) {
 
 	ctx, span := d.tracer.Start(ctx, "SQL SELECT", trace.WithSpanKind(trace.SpanKindClient))
 	span.SetAttributes(
-		semconv.PeerServiceKey.String("mysql"),
```

Comment:
I don't understand your thought process behind these changes. Why is it appropriate to remove this attribute? Just because it happens to help your specific graph implementation doesn't make it the right thing to do.

Reply:
Hi @yurishkuro, I apologize for making the above changes. A problem was identified in the code where the dependency links were created, which did not contain a check to prevent the addition of parent services with the same name as child services. The new changes have been added. Could you please review them?